WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryItems/HistoryItemViewModelBase.cs

```diff
@@ -55,11 +56,13 @@ protected HistoryItemViewModelBase(TransactionModel transaction)
 			.Subscribe();
 	}
 
-	protected HistoryItemViewModelBase(UiContext uiContext, TransactionModel transaction) : this(transaction)
+	protected HistoryItemViewModelBase(IWalletModel wallet, UiContext uiContext, TransactionModel transaction) : this(wallet, transaction)
 	{
 		UiContext = uiContext;
 	}
 
+	protected IWalletModel Wallet { get; }
```

Comment:
In the UI code fields are preferred. Using a property implies that it is used in the View for binding.

Reply:
I have a warning when using a field